auslib/db.py

```diff
@@ -2648,7 +2679,7 @@ def setupChangeMonitors(self, relayhost, port, username, password, to_addr, from
                                                                to_addr,
                                                                from_addr,
                                                                use_tls)
-        self.releases.onUpdate = read_only_bleeter
+        self.releasesReadonly.onUpdate = read_only_bleeter
```

Comment:
Hmm, if we're only doing insert's and delete's on this table, what do we get from this ? Maybe we should just notify on all changes in releasesReadonly, they should be fairly infrequent even after bug 1256719 (which will lock each release after all information is submitted into Balrog for it).